docker/docker-compose-servers.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3.7"
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    #logging:
```

Comment:
Why do we comment out the logging-driver line instead of just removing it

Reply:
Damn, I've commited a lot of local changes I used for testing, sorry about this. Should be reverted now.